components/EligibleCollections/EligibleCollections.tsx

```diff
@@ -0,0 +1,131 @@
+import { ethers } from 'ethers';
+import { useAsyncValue } from 'hooks/useAsyncValue';
+import { useConfig } from 'hooks/useConfig';
+import {
+  OracleInfoProvider,
+  useOracleInfo,
+} from 'hooks/useOracleInfo/useOracleInfo';
+import { useTarget } from 'hooks/useTarget';
+import { SECONDS_IN_A_YEAR } from 'lib/constants';
+import { formatPercent } from 'lib/numberFormat';
+import { OraclePriceType } from 'lib/oracle/reservoir';
+import { SubgraphController } from 'lib/PaprController';
+import { subgraphControllerByAddress } from 'lib/pAPRSubgraph';
+import { percentChange } from 'lib/tokenPerformance';
+import dynamic from 'next/dynamic';
+import { useMemo } from 'react';
+import Marquee from 'react-fast-marquee';
+
+import { Collection } from './Collection';
+import { Loader } from './ConnectWallet';
+import { COLLECTIONS } from './constants';
+import styles from './EligibleCollections.module.css';
+
+export function EligibleCollections() {
+  const collections = useMemo(() => COLLECTIONS.map((c) => c.address), []);
+  return (
+    <OracleInfoProvider collections={collections}>
+      <EligibleCollectionsInner />
+    </OracleInfoProvider>
+  );
+}
+
+export function EligibleCollectionsInner() {
+  const config = useConfig();
+  const subgraphData = useAsyncValue(
+    () => subgraphControllerByAddress(config.controllerAddress, 'paprMeme'),
+    [config],
+  );
+  const oracleInfo = useOracleInfo(OraclePriceType.twap);
+  const targetResult = useTarget();
+
+  return (
+    <div className={styles.wrapper}>
+      <APR paprController={subgraphData?.paprController} />
+      <Marquee className={styles.marquee} pauseOnHover gradient={false}>
+        {COLLECTIONS.map((c) => (
+          <Collection
+            paprController={subgraphData?.paprController}
+            oracleInfo={oracleInfo}
```

Comment:
out of curiosity, why prop drill these rather than use the hooks directly within the `Collection` component?

Reply:
No solid reason, artifact of earlier prototyping. Next time we're in here we can move it into the component